library/src/main/assets/google_map.html

```diff
@@ -90,6 +92,26 @@
       icon: 'http://maps.google.com/mapfiles/ms/icons/red-dot.png'
   });
 
+  if (title == "null" && snippet == "null") {
+        <!-- Title and snippet not set, don't create default InfoWindow -->
+  } else if (title != "null" && snippet == "null") {
```

Comment:
@nwadams Well to emulate the native map exactly, we don't want to create an InfoWindow if no title and snippet is set. That is why I wrote the logic that way. With your suggestion, an infoWindow will be created every time, unless we change the logic in the showDefaultInfoWindow function.